howso/client/base.py

```diff
@@ -339,30 +309,27 @@ def react_into_features(
         """Calculate conviction and other data for the specified feature(s)."""
 
     @abstractmethod
-    def react_into_trainee(
+    def react_aggregate(
         self, trainee_id, *,
         action_feature=None,
         confusion_matrix_min_count=None,
         context_features=None,
-        contributions=None,
-        contributions_robust=None,
+        details=None,
+        feature_influences_action_feature=None,
         hyperparameter_param_path=None,
-        mda=None,
-        mda_permutation=None,
-        mda_robust=None,
-        mda_robust_permutation=None,
+        num_samples=None,
         num_robust_influence_samples=None,
         num_robust_residual_samples=None,
         num_robust_influence_samples_per_case=None,
-        num_samples=None,
-        residuals=None,
-        residuals_robust=None,
+        prediction_stats_action_feature=None,
+        residuals_hyperparameter_feature=None,
+        robust_hyperparameters=None,
         sample_model_fraction=None,
         sub_model_size=None,
-        use_case_weights=False,
+        use_case_weights=None,
         weight_feature=None,
-    ):
-        """Compute and cache specified feature interpretations."""
+    ) -> Union["DataFrame", dict]:
```

Comment:
```suggestion
    ) -> Union[DataFrame, dict]:
```

or, if you add `from __future__ import annotations` to the top of the file you can,


```suggestion
    ) -> DataFrame | dict:
```